logic/expiry_logic.go

```diff
@@ -6,8 +6,10 @@ import (
 	"url-shortner/storage/dao"
 )
 
+const _deleteQuery = "DELETE from redirection WHERE expiry < NOW()"
```

Comment:
Why it's a constant, not a variable?

Reply:
This delete query should not change over the course of the running App hence a constant

Reply:
Okay, bro. 
JFYI: Constant are always written in Upper case 🙂

Reply:
I use this styleguide for reference https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_